_sections/home-english.md

```diff
@@ -490,12 +489,12 @@ In the next example, below, we use relevant syntax for a **select_multiple** que
 
 Since the pizza topping question allows multiple responses, we have to use the ``selected(${favorite_topping}, 'cheese')`` expression, because we want the cheese question to appear every time the user selects **cheese** as one of the answers (regardless of whether additional answers are selected).
 
-Earlier we mentioned there was an alternative method for specifying other for multiple-choice questions which is more appropriate for large-scale surveys. This can be done using the same relevant syntax from the example above:
+Earlier we mentioned that relevance could be used to specify a free-text "other" option for multiple-choice questions. The advantages of doing this over using the **or_other** shortcut are that you get control over the choice name, you get to specify the text that the user sees, and it works with multi-language forms. Here is an example:
```

Comment:
Suggest replacing "Earlier we mentioned" with "In the [Specify Other](https://xlsform.org/en/#specify-other) section it is mentioned".